src/docs/reference/modules/OpenId/README.md

```diff
@@ -49,9 +49,31 @@ Available settings are:
 - Enable Authorization Endpoint.
 - Enable Logout Endpoint.
 - Enable User Info Endpoint.
-- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3>
+- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3> \
+   Request sample:
+    ``` json 
+    //EndPoint="/connect/token" ,Method=POST,Content-Type=application/x-www-form-urlencoded 
+    {
```

Comment:
The token request MUST be formatted using `x-www-form-urlencoded` so it's kinda confusing to use JSON here (the token endpoint returns JSON but doesn't accept JSON inputs).

Reply:
Yes, I understand what you mean, I should use a JavaScript script as an example

Reply:
@kevinchalet  I have updated the sample codes